webpack.config.js

```diff
@@ -182,6 +182,7 @@ module.exports = env => {
             "setImmediate": false,
             "fs": "empty",
             "__dirname": false,
+            "child_process": "empty",
```

Comment:
What is this for?

Reply:
I saw this on readme file. That way I add that here.